pyproject.toml

```diff
@@ -56,7 +56,6 @@ PyYAML = "^6.0"
 cognite-sdk = {version = ">=7.13.8", extras = ["pandas"]}
```

Comment:
Don't we want this to be >=7.37?

Reply:
Good point, I just updated the lock file :-D I have pinned now.